src/briefcase/__main__.py

```diff
@@ -10,6 +10,7 @@ def main():
     command = None
     try:
         command, options = parse_cmdline(sys.argv[1:])
+        log = command.logger
```

Comment:
It took me a moment to work out what was going on here. I think this points at an architectural flaw; we should be passing `log` to `parse_cmdline`, and using that log instance, rather than capturing a replacement log instance. However, that's a much more involved piece of dental work that doesn't need to block this PR.